src/widgets/FileWidget/RemoveHandle.js

```diff
@@ -38,6 +47,7 @@ RemoveHandle.propTypes = {
 
 RemoveHandle.defaultProps = {
   removeStyle: null,
+  auto: false,
```

Comment:
You can't switch the default to `auto: false`. It will have consequences on the `FileWidget` with `schema.type = 'array'`.